packages/ra-data-graphql/src/buildApolloClient.js

```diff
@@ -0,0 +1,18 @@
+import { ApolloClient, createNetworkInterface } from 'apollo-client';
```

Comment:
rename to `buildApolloProvider`

Reply:
Nope, it builds an Apollo client which will be used by the data provider